vpr/src/base/vpr_types.h

```diff
@@ -900,6 +901,34 @@ class GridBlock {
     vtr::NdMatrix<t_grid_blocks, 3> grid_blocks_;
 };
 
+class PlaceLocVars {
```

Comment:
What do you think about `BlockLocationRegistry`?

Reply:
I like that one a lot! Either that or the shorter version that Vaughn sent below `BlkLocRegistery`. Very good name in my opinion because it captures what is being done well.